followthemoney/cli/ocds.py

```diff
@@ -82,7 +85,7 @@ def convert_release(release):
 
     tender = release.pop('tender', {})
     contract = model.make_entity('Contract')
-    contract.make_id(release.pop('id', None))
+    contract.make_id(release.pop('ocid', None))
```

Comment:
Should we play more of the "if you have X, use X, otherwise use Y" game here?

Reply:
I have to consult the source data here (concerned about possible dupes).